### ISSUE TITLE
Crash on startup/GetEPGForChannel

### ISSUE THREAD

**Issue by appleonkel** [flubshi/pvr.waipu#1]
On Libreelec 9.0.1 it crashes on syncing the EPG Channel data at Nickelodeon, since today.

Here is the relevant crash.log:
```
Thread 1 (Thread 0x5c53e370 (LWP 1359)):
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
#1  0x74e10184 in __GI_abort () at abort.c:79
#2  0x74d278b0 in __gnu_cxx::__verbose_terminate_handler() () from /usr/lib/libstdc++.so.6
#3  0x74d255ec in ?? () from /usr/lib/libstdc++.so.6
#4  0x74d25660 in std::terminate() () from /usr/lib/libstdc++.so.6
#5  0x74d259c8 in __cxa_throw () from /usr/lib/libstdc++.so.6
#6  0x74d4e7b0 in std::__throw_invalid_argument(char const*) () from /usr/lib/libstdc++.so.6
#7  0x6ef0372c in std::stoi(std::string const&, unsigned int*, int) () from /storage/.kodi/addons/pvr.waipu/pvr.waipu.so.0.0.9
#8  0x6ef01748 in WaipuData::GetEPGForChannel(ADDON_HANDLE_STRUCT*, PVR_CHANNEL const&, long, long) () from /storage/.kodi/addons/pvr.waipu/pvr.waipu.so.0.0.9
#9  0x6eefe0a4 in GetEPGForChannel () from /storage/.kodi/addons/pvr.waipu/pvr.waipu.so.0.0.9
#10 0x006622f0 in ?? ()
#11 0x00659b64 in PVR::CPVRClient::DoAddonCall(char const*, std::function<PVR_ERROR (KodiToAddonFuncTable_PVR const*)>, bool, bool) const ()
#12 0x00663090 in PVR::CPVRClient::GetEPGForChannel(std::shared_ptr<PVR::CPVRChannel> const&, PVR::CPVREpg*, long, long, bool) ()
#13 0x0056dfbc in PVR::CPVREpg::UpdateFromScraper(long, long, bool) ()
#14 0x00587834 in PVR::CPVREpg::LoadFromClients(long, long, bool) ()
#15 0x00587fe8 in PVR::CPVREpg::Update(long, long, int, bool) ()
#16 0x0058af20 in PVR::CPVREpgContainer::UpdateEPG(bool) ()
#17 0x0058bf48 in PVR::CPVREpgContainer::Process() ()
#18 0x0076c3f8 in CThread::Action() ()
#19 0x00773914 in CThread::staticThread(void*) ()
#20 0x76f37338 in start_thread (arg=0x5c53e370) at pthread_create.c:486
Backtrace stopped: Cannot access memory at address 0x64f8
############# END STACK TRACE ###############
```

**Comment by flubshi** [comment]
Thank you for reporting this issue. It shoud be fixed in cdf73819ce96521132bc6122e038a41e0f7bedd1

Now we have to figure out whether LibreELEC takes the latest commit of this plugin or if we have to bump it..

**Comment by appleonkel** [comment]
I built your last commit. It works again, now.
Thank you!
